PENDING_RELEASE_NOTES.md

```diff
@@ -2,6 +2,9 @@
 ## Version 15.15.2
```

Comment:
If a new feature, should probably update the "middle" number instead of the last